mitmproxy/tools/console/master.py

```diff
@@ -123,12 +123,30 @@ def get_editor(self) -> str:
         else:
             return "vi"
 
+    def get_hex_editor(self) -> str:
+        if m := os.environ.get("MITMPROXY_EDITOR"):
+            return m
+        if m := os.environ.get("EDITOR"):
+            return m
+        for editor in "hexedit", "hexyl":
+            if shutil.which(editor):
+                return editor
+        if os.name == "xxd":
+            return "xxd"
+        else:
+            return "ghex"
+
     def spawn_editor(self, data):
+        if strutils.is_mostly_bin(data):
+            isBinary = True
```

Comment:
- Please use `snake_case` for Python variables.
- Is there a reason why you create a variable here instead of directly using `strutils.is_mostly_bin` later on?

Reply:
I got it! Thanks.